csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -158,6 +158,53 @@ private HashSet<string> AddFrameworkDlls(HashSet<string> dllPaths)
         {
             var frameworkLocations = new HashSet<string>();
 
+            var frameworkReferences = Environment.GetEnvironmentVariable(EnvironmentVariableNames.DotnetFrameworkReferences);
+            var frameworkReferencesUseSubfolders = Environment.GetEnvironmentVariable(EnvironmentVariableNames.DotnetFrameworkReferencesUseSubfolders);
+            _ = bool.TryParse(frameworkReferencesUseSubfolders, out var useSubfolders);
+            if (!string.IsNullOrWhiteSpace(frameworkReferences))
+            {
+                RemoveFrameworkNugetPackages(dllPaths);
+                RemoveNugetPackageReference(FrameworkPackageNames.AspNetCoreFramework, dllPaths);
+                RemoveNugetPackageReference(FrameworkPackageNames.WindowsDesktopFramework, dllPaths);
```

Comment:
I think this doesn't work on some early .net core variants, where the framework was split up into many nuget packages, and therefore they wouldn't be identified as framework references but as standard nuget packages.